src/web/utils/FocusManager.ts

```diff
@@ -108,7 +106,11 @@ export class FocusManager extends FocusManagerBase {
     static focusFirst(last?: boolean) {
         const focusable = Object.keys(FocusManager._allFocusableComponents)
             .map(componentId => FocusManager._allFocusableComponents[componentId])
-            .filter(storedComponent => !storedComponent.removed && !storedComponent.restricted && !storedComponent.limitedCount)
+            .filter(storedComponent =>
+                !storedComponent.removed &&
+                !storedComponent.restricted &&
+                !storedComponent.limitedCount &&
```

Comment:
limitedCount and limitedCountAccessible are not booleans, so the proper check here is "> 0" - for readability.